### PR TITLE
Add constituency info to signature signed page

### PR DESCRIPTION
**Pull request by davidbasalla**
For the constituency info to appear on the signed page, I had to enable member
routes for the signed signature to access the signature's postcode. Eager to hear
alternative/better approaches of how to achieve this!

### CODE REVIEW

File: app/controllers/signatures_controller.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class SignaturesController < ApplicationController
   before_filter :retrieve_petition, :only => [:new, :create, :thank_you, :signed]
+  before_filter :retrieve_signature, :only => [:signed]
```
Comment (otlaitil) [inline app/controllers/signatures_controller.rb]: You could probably use this new ```before_filter``` in other actions in the controller as well (```verify``` and ```retrieve_signature``` at least)

File: spec/controllers/signatures_controller_spec.rb [hunk 2]
```diff
@@ -5,6 +5,16 @@
     context "signature of user who is not the petition's creator" do
       let(:petition) { FactoryGirl.create(:petition) }
       let(:signature) { FactoryGirl.create(:pending_signature, :petition => petition) }
+      let(:api_url) { "http://data.parliament.uk/membersdataplatform/services/mnis/Constituencies" }
+      let(:fake_body) {
+        "<Constituencies>
```
Comment (otlaitil) [inline spec/controllers/signatures_controller_spec.rb]: Just wondering if our super cool ConstituencyApi could have some test helper to generate these mock responses for us & maybe help to stub the requests? Just an idea. 

At least we could use ```let(:api_url) { ConstituencyApi::Client.API_URL }```

**Comment by otlaitil** [comment]
LGTM :+1:

**Comment by alanth** [comment]
LGTM. Merging

**Pull request merged**